src/transformers/utils/auto_docstring.py

```diff
@@ -46,7 +46,10 @@
 PLACEHOLDER_TO_AUTO_MODULE = {
     "image_processor_class": ("image_processing_auto", "IMAGE_PROCESSOR_MAPPING_NAMES"),
     "video_processor_class": ("video_processing_auto", "VIDEO_PROCESSOR_MAPPING_NAMES"),
-    "feature_extractor_class": ("feature_extraction_auto", "FEATURE_EXTRACTOR_MAPPING_NAMES"),
+    "feature_extractor_class": (
+        "feature_extraction_auto",
+        "FEATURE_EXTRACTOR_MAPPING_NAMES",
+    ),
```

Comment:
Hey, there are a lot of unrelated style changes here! If your style tools are doing this, make sure to `pip install -e .[quality]` to use the same versions as the codebase. Then try reverting these changes and ping me to take a look - it'll be a lot easier to review when it's just the actual change.